h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -3027,7 +3027,12 @@ public double applyBounds(double d, int i) {
     }
 
     private Frame encodeCategoricalsIfPresent(Frame beta_constraints) {
-      FrameUtils.BetaConstraintsEncoder constraintsEncoder = new FrameUtils.BetaConstraintsEncoder(_dinfo.coefNames(), _dinfo.coefOriginalNames());
+      Log.debug("Encoding beta constraints...");
+      String[] coefNames = _dinfo.coefNames();
+      Log.debug("_dinfo.coefNames() content: " + Arrays.toString(coefNames));
```

Comment:
how often is this function called? if the number is non-trivial it would be good to prefix the Arrays.toString calls with a check for debug being enabled

Reply:
rather will prefix

Reply:
33c8112